README.md

```diff
@@ -197,14 +195,94 @@ connection.
 Be aware that using serial printing can affect the timing of the firmware. You may experience
 strange behavior while printing that does not occur otherwise.
 
+# Controlling Your Vehicle - an Example Application
+
+Now that all your Arduino modules are properly setup, it is time to start sending control commands.
+We've created an example application, joystick commander, that uses the OSCC API to interface with the firmware, allowing you to send commands using a game controller and receive reports from the on-board OBD-II CAN. These commands are converted into CAN messages, which the OSCC API sends to the respective Arduino modules and are used to actuate the vehicle.
+
+[OSCC Joystick Commander](https://github.com/PolySync/oscc-joystick-commander)
+
+# OSCC API
+
+**Open and close CAN channel to OSCC Control CAN.**
+
+```c
+oscc_result_t oscc_open( uint channel )
+oscc_result_t oscc_close( uint channel )
+```
+
+These methods are the start and end points of using the OSCC API in your application. ```oscc_open``` will open a socket connection
+on the specified CAN channel, enabling it to quickly receive reports from and send commands to the firmware modules.
+When you are ready to terminate your application, ```oscc_close``` can terminate the connection.
 
-## Tests
+**Enable and disable all OSCC modules.**
+
+```c
+oscc_result_t oscc_enable( void )
+oscc_result_t oscc_disable( void )
+```
+
+After you have initialized your CAN connection to the firmware modules, these methods can be used to enable or disable the system. This
+allows your application to choose when to enable sending commands to the firmware. Although you can only send commands when the system is
+enabled, you can receive reports at any time.
+
+**Publish control command to the corresponding module.**
+
+```c
+oscc_result_t publish_brake_position( double normalized_position )
```

Comment:
When viewing the document, the first statement in these code blocks is syntax highlighted but the others are not, perhaps markdown is looking for a terminating `;` character?

Reply:
I noticed that and didn't know how to fix it. Good idea with the semicolons, they worked.

f069c76